tests/by-util/test_tr.rs

```diff
@@ -1413,3 +1413,14 @@ fn check_complement_1_unique_in_set2() {
         .args(&["-c", "[:upper:]", arg.as_str()])
         .succeeds();
 }
+
+#[test]
+fn check_complement_set2_too_big() {
+    let x231 = "x".repeat(231);
```

Comment:
The existence of this test is great! Can you also add a test to make sure that using 230 works as intended?